parquet-common/src/main/java/org/apache/parquet/hadoop/metadata/CompressionCodecName.java

```diff
@@ -30,7 +30,8 @@ public enum CompressionCodecName {
   LZO("com.hadoop.compression.lzo.LzoCodec", CompressionCodec.LZO, ".lzo"),
   BROTLI("org.apache.hadoop.io.compress.BrotliCodec", CompressionCodec.BROTLI, ".br"),
   LZ4("org.apache.hadoop.io.compress.Lz4Codec", CompressionCodec.LZ4, ".lz4"),
-  ZSTD("org.apache.parquet.hadoop.codec.ZstandardCodec", CompressionCodec.ZSTD, ".zstd");
+  ZSTD("org.apache.parquet.hadoop.codec.ZstandardCodec", CompressionCodec.ZSTD, ".zstd"),
+  LZ4_RAW("org.apache.parquet.hadoop.codec.Lz4RawCodec", CompressionCodec.LZ4_RAW, ".lz4raw");
```

Comment:
I think both file extensions for LZ4 and LZ4RAW should be removed.
The reason is that the `lz4` command-line utilities expect the LZ4 frame format.
In any case, the `.lz4` extension should certainly not be used for the "Hadoop LZ4" compression codec.

Reply:
I searched the parquet-mr repo and found the only usage of the codec extension as below: 

https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java#L420
```java
  public RecordWriter<Void, T> getRecordWriter(TaskAttemptContext taskAttemptContext, Mode mode)
      throws IOException, InterruptedException {

    final Configuration conf = getConfiguration(taskAttemptContext);

    CompressionCodecName codec = getCodec(taskAttemptContext);
    String extension = codec.getExtension() + ".parquet";
    Path file = getDefaultWorkFile(taskAttemptContext, extension);
    return getRecordWriter(conf, file, codec, mode);
  }
```

The extension is part of the enum definition and cannot be removed. I propose to rename `lz4` to `lz4hadoop` to explicitly differentiate this lz4 variant from standard `lz4`.  @pitrou

Reply:
Sounds good to me.